pkgs/development/python-modules/pymedio/default.nix

```diff
@@ -0,0 +1,40 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pythonOlder
+, pytestCheckHook
+, numpy
+}:
+
+buildPythonPackage rec {
+  pname = "pymedio";
+  version = "0.2.13";
+  disabled = pythonOlder "3.9";
+
+  src = fetchFromGitHub {
+    owner = "jcreinhold";
+    repo = "pymedio";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-iHbClOrtYkHT1Nar+5j/ig4Krya8LdQdFB4Mmm5B9bg=";
+  };
+
+  # relax Python dep to work with 3.10.x
+  postPatch = ''
+    substituteInPlace setup.cfg --replace "!=3.10.*," ""
+  '';
+
+  propagatedBuildInputs = [ numpy ];
+
+  doCheck = false;  # requires SimpleITK python package (not in Nixpkgs)
```

Comment:
What about https://github.com/NixOS/nixpkgs/blob/620eac54d5194d754ff2170e3634d0c3c1afd113/pkgs/development/libraries/simpleitk/default.nix ?

Reply:
It requires the Python bindings to be built, which is not obvious how to do (but I will certainly attempt at some point with a fast build machine).